libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -80,20 +80,25 @@ string YulUtilFunctions::splitExternalFunctionIdFunction()
 	});
 }
 
-string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata)
+string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata, bool _with_cleanup)
 {
-	string functionName = "copy_" + string(_fromCalldata ? "calldata" : "memory") + "_to_memory";
+	string functionName =
+		"copy_"s +
+		(_fromCalldata ? "calldata"s : "memory"s) +
+		"_to_memory"s +
+		(_with_cleanup ? "_cleanup"s : ""s);
```

Comment:
Haha, My [previous suggestion](https://github.com/ethereum/solidity/pull/13282#discussion_r937004620) was about the function name, not the parameter :)
```suggestion
		(_cleanup ? "_with_cleanup"s : ""s);
```
Since we're not releasing today after all, we could still change that but on the other hand it's not such a big deal... I just thought that the name would be more readable that way.